VirtoCommerce.Platform.Web/Scripts/common/directives/chars-count.js

```diff
@@ -0,0 +1,28 @@
+﻿angular.module('platformWebApp')
+    .directive('vaCharsCount', function () {
+        return {
+            template: `<div class="chars-counter">
+                           <span>{{vaField == undefined ? '0' : vaField.length}}</span> / <span>{{vaMaxlength}}</span> 
```

Comment:
`vaField == undefined` can be replaced to `vaField ? ` or `if (vaField)` or better `!!vaField`